packages/jest-types/__typetests__/globals.test.ts

```diff
@@ -135,6 +135,24 @@ expectType<void>(test.skip.each(list)(function named() {}, fn));
 expectType<void>(test.skip.each(list)(class {}, fn));
 expectType<void>(test.skip.each(list)(class Named {}, fn));
 
+expectType<void>(test.failing(123, fn));
+expectType<void>(test.failing(() => {}, fn));
+expectType<void>(test.failing(function named() {}, fn));
+expectType<void>(test.failing(class {}, fn));
+expectType<void>(test.failing(class Named {}, fn));
+
+expectType<void>(test.skip.failing(123, fn));
```

Comment:
Hm.. Are `test.skip.failing` and simply `test.skip` somehow different in behaviour? Question is not about the test. I try to figure out if the implementation necessary?

Reply:
Just saw your comment. Thanks

Reply:
Happy to hear that :D